src/python/pants/build_graph/build_graph.py

```diff
@@ -127,7 +127,7 @@ def reset(self):
     """
     self._target_by_address = OrderedDict()
     self._target_dependencies_by_address = defaultdict(OrderedSet)
-    self._target_dependees_by_address = defaultdict(set)
+    self._target_dependees_by_address = defaultdict(OrderedSet)
```

Comment:
Weird we used `set` here when we used `OrderedSet` right above. Looks like a good change.